blogposts/release-anxiety.md

```diff
@@ -2,7 +2,7 @@
 title: "Let's talk about release anxiety"
 description: 'Engineers are caught in a vise between two pressures: the pressure to get code changes into the next release and to protect the release from bugs. The result? Release anxiety.'
 externalTitle: 'Release anxiety: What it is and how to address it'
-externalDescription: 'Release anxiety plagues many engineers. Can you squeeze in this last code change? Will this last code change break production? Let's talk about ways to deal with the anxiety.'
+externalDescription: "Release anxiety plagues many engineers. Can you squeeze in this last code change? Will this last code change break production? Let's talk about ways to deal with the anxiety."
```

Comment:
@nickmyyz We've seen this issue before – there was an apostrophe in this line causing the build to break because it didn't have a corresponding one. I've fixed it by enclosing everything in double quotation marks!

Reply:
@rebeccadee dang, I was close! I fixed that in the title but didn't see the apostrophe in the external. Thank you! :)